lib/ansible/modules/cloud/ovirt/ovirt_vm.py

```diff
@@ -363,6 +363,11 @@
         type: bool
         version_added: "2.5"
         aliases: [ 'sysprep_persist' ]
+    kernel_params_persist:
+        description:
+            - "If I(true) kernel_params, initrd_path and kernel_path will be set to Linux Boot Options."
```

Comment:
C(kernel_params), C(initrd_path) C(kernel_path)

Reply:
"If I(true) C(kernel_params), C(initrd_path) and C(kernel_path) will persist in virtual machine configuration, if I(False) it will be used for run once."